src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala

```diff
@@ -100,9 +99,10 @@ trait EmbeddedServiceOrchestrator extends ResourceProvider with StartAndStopServ
 
     val environment: Environment = Environment.simple(mode = applicationMode)
     val application: Application = new GuiceApplicationBuilder(environment = environment, configuration = config).build()
+    Play.start(application)
```

Comment:
No. `NettyServer` takes care of this when it is stopped, which happens in uk.gov.hmrc.play.it.EmbeddedServiceOrchestrator#stop